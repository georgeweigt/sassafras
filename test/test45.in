* From NIST
* http://www.itl.nist.gov/div898/
* handbook/prc/section4/prc436.htm#example1 ;

data ;
input trt $ y @@ ;
datalines ;
A 6.9 A 5.4 A 5.8 A 4.6 A 4.0
B 8.3 B 6.8 B 7.8 B 9.2 B 6.5
C 8.0 C 10.5 C 8.1 C 6.9 C 9.3
D 5.8 D 3.8 D 6.1 D 5.6 D 6.2
;
proc anova ;
model y = trt ;
means trt / lsd ;
