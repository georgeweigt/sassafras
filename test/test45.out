                              Analysis of Variance

    Source     DF     Sum of Squares     Mean Square     F Value     Pr > F
    Model       3        38.82000000     12.94000000        9.72     0.0007
    Error      16        21.29200000      1.33075000                       
    Total      19        60.11200000                                       

                R-Square     Coeff Var     Root MSE       Y Mean
                0.645795     17.014475     1.153581     6.780000

      Source     DF        Anova SS     Mean Square     F Value     Pr > F
      TRT         3     38.82000000     12.94000000        9.72     0.0007

                                 Mean Response

              TRT     N       Mean Y     95% CI MIN     95% CI MAX
              A       5     5.340000       4.246347       6.433653
              B       5     7.720000       6.626347       8.813653
              C       5     8.560000       7.466347       9.653653
              D       5     5.500000       4.406347       6.593653

                       Least Significant Difference Test

  TRT    TRT      Delta Y    95% CI MIN    95% CI MAX    t Value    Pr > |t|  
  A      B      -2.380000     -3.926659     -0.833341      -3.26      0.0049 *
  A      C      -3.220000     -4.766659     -1.673341      -4.41      0.0004 *
  A      D      -0.160000     -1.706659      1.386659      -0.22      0.8292  
  B      A       2.380000      0.833341      3.926659       3.26      0.0049 *
  B      C      -0.840000     -2.386659      0.706659      -1.15      0.2665  
  B      D       2.220000      0.673341      3.766659       3.04      0.0078 *
  C      A       3.220000      1.673341      4.766659       4.41      0.0004 *
  C      B       0.840000     -0.706659      2.386659       1.15      0.2665  
  C      D       3.060000      1.513341      4.606659       4.19      0.0007 *
  D      A       0.160000     -1.386659      1.706659       0.22      0.8292  
  D      B      -2.220000     -3.766659     -0.673341      -3.04      0.0078 *
  D      C      -3.060000     -4.606659     -1.513341      -4.19      0.0007 *

